tartiflette/types/exceptions/__init__.py

```diff
@@ -0,0 +1 @@
+from tartiflette.types.exceptions.tartiflette import *
```

Comment:
I would rather avoid star imports.

* This is not explicit but implicit
* It can shadow some other variables
* Break support with some interesting feature of code linting

This is not pre'm not a big fan of star imports. It is

Reply:
I'm not a big fan of this either and I must admit I did this to know about your position regarding simplifying exceptions imports?

Now two options remain:
1. I just import `GraphQLError` as it is most likely the one users will want to use
2. I import everything explicitly but we'll have to make sure not to forget to add them shall we add new ones

I initially went for 1/ but I'd like your point of view plz

Reply:
Yep, please, no * ;).
Import everything you need, and only the thing you need. 
Here it's a little bit painfull but nonetheless, it's what should be done.

As this is an 'exception' module, if you add one, you'll see quickly that you forgot it in the init, cause you'll have to import it in ordee to use it.

This module is for internal exceptions that the engine uses for it's own purpose. 

I'm thinking for quite some time to extract 'GraphQLError' into a more easily importable location as it should be the only 'public' exception used/derived by ttftt users.

Reply:
done